sharing_portal/templates/sharing_portal/edit.html

```diff
@@ -37,7 +37,13 @@ <h2 class="artifactTitle">Edit artifact</h2>
 
   <div class="layoutGrid__side">
     <div class="artifactVersions">
-      <h4><i class="fa fa-files-o"></i> Versions</h4>
+      <div style="display: flex">
```

Comment:
Please give this a class and do this in CSS as opposed to inline styles. Inline styles can get out of hand and have a habit of building on themselves over time.